misk-policy/README.md

```diff
@@ -0,0 +1,69 @@
+# misk-policy
+
+The misk-policy module provides a type safe interface to policy engine side cars.
+The only supported policy engine is [Open Policy Agent (OPA)](https://www.openpolicyagent.org/docs/latest/) today.
+
+## Assumptions
+
+Setup of OPA is not included.  
+The policy modules assumes that the policy engine is accessible via HTTP(s).
```

Comment:
Since HTTP is the transport layer. This would be more accurately, `is accessible via a TCP socket or abstract unix domain socket`